### ISSUE TITLE
Cargo audit failure due to tungstenite dependency

### ISSUE THREAD

**Issue by rageshkrishna** [omjadas/hudsucker#89]
I get the following [high severity audit failure](https://rustsec.org/advisories/RUSTSEC-2023-0065) due to hudsucker's dependency on a vulnerable version of tungstenite:

```
Crate:     tungstenite
Version:   0.19.0
Title:     Tungstenite allows remote attackers to cause a denial of service
Date:      2023-09-25
ID:        RUSTSEC-2023-0065
URL:       https://rustsec.org/advisories/RUSTSEC-2023-0065
Severity:  7.5 (high)
Solution:  Upgrade to >=0.20.1
Dependency tree:
tungstenite 0.19.0
├── tokio-tungstenite 0.19.0
│   ├── hyper-tungstenite 0.10.0
│   │   └── hudsucker 0.20.0
│   │       └── <my component>
│   └── hudsucker 0.20.0
└── hyper-tungstenite 0.10.0

error: 1 vulnerability found!
```

It seems like the latest release of [hyper-tungstenite](https://crates.io/crates/hyper-tungstenite/0.12.0/dependencies) uses a fixed version of tungstenite, so it would be good if hudsucker could be updated to resolve the security issue.

**Comment by omjadas** [comment]
I have just published [v0.21.0](https://github.com/omjadas/hudsucker/releases/tag/v0.21.0), which should fix this.

**Comment by rageshkrishna** [comment]
Thank you, @omjadas! I somehow completely missed the notification for this update.
